src/algebra/lie_algebra.lean

```diff
@@ -409,39 +441,120 @@ instance lie_subalgebra_lie_algebra (L' : lie_subalgebra R L) :
     @lie_algebra R L' _ (lie_subalgebra_lie_ring _ _ _) :=
 { lie_smul := by { intros, apply set_coe.ext, apply lie_smul } }
 
+@[simp] lemma lie_subalgebra.mem_coe {L' : lie_subalgebra R L} {x : L} :
```

Comment:
Please also give a simp-lemma `mem_coe'` for the coercion to submodule.

Reply:
Done!